### PR TITLE
Add Podman 3 and Docker Compose video

### PR DESCRIPTION
**Pull request by Fodoj**

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Thanks @Fodoj !

**Comment by Fodoj** [comment]
I might have made a mistake somewhere again, post is not visible on the website :)

File: _posts/2021-05-26-new.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Dockerless: Build and Run Containers with Podman and Systemd
```
Comment (TomSweeneyRedHat) [inline _posts/2021-05-26-new.md:2]: Missed this earlier.  @Fodoj could you put a new PR together with a fix to the right title?  The colon `:` here is keeping it from being published.
`title: Podman 3 and Docker Compose - How Does the Dockerless Compose Work?`

Comment (Fodoj) [reply _posts/2021-05-26-new.md:2]: fixed here - https://github.com/containers/podman.io/pull/407

fyi, links to 3.1 release are giving 404 -  https://podman.io/new/2021/04/02/new.html -> https://podman.io/releases/2020/04/02/podman-release-v3.1.0.html

**Comment by TomSweeneyRedHat** [comment]
@Fodoj I just fixed the 3.1 link, it was trying to point to 2020 instead of 2021 as it should have been.  Thanks for the heads up.

**Pull request merged**